public/js/index.js

```diff
@@ -3,6 +3,6 @@ const getStartedButton = document.getElementById("getStarted");
 /**
  * Adds event listener for "Get Started" button and redirects to /webcam page
  */
-getStartedButton.addEventListener("click", () => {
+getStartedButton.onclick = () => {
```

Comment:
Just wanted to double-check a couple of things with you:

- So if you do Command Shift F and search for `addEventLIstener` there's no more?
- Also did you notice for each page whether there was any behaviour that was broken as a result of this change? I think `track.js` is the most fragile and susceptible to breaking changes. Does every change you made still work as expected?
  - the first time you click the button or html element it should perform as expected. But what I'm concerned about is the second third fourth time etc.

Reply:
Yup! I've covered all our files and none of the behaviour was broken either (with multiple clicks included)!

Reply:
NOICEEEEEE perfect thanks for doing this